Tests/TLSEvaluationTests.swift

```diff
@@ -165,8 +146,8 @@ class TLSEvaluationExpiredLeafCertificateTestCase: BaseTestCase {
         // Then
         XCTAssertNotNil(error, "error should not be nil")
 
-        if let error = error as? AFError {
-            XCTAssertTrue(error.isCertificatePinningError, "should be .certificatePinningFailed")
+        if let error = error?.asAFError {
+            XCTAssertTrue(error.isServerTrustEvaluationError, "should be .certificatePinningFailed")
```

Comment:
I kept them where I could to serve as a reminder we should probably be testing these scenarios for specific errors. I can remove them if you want but I think we'll want something like them at some point. Once everything is merged and we're into the beta process I'd like to evaluate test coverage and ensure we're looking at specific error scenarios, since that's where we're missing a lot of coverage I think.

Reply:
Sounds good @jshier 👌🏻